src/AppBundle/Entity/Repository/DepartmentRepository.php

```diff
@@ -30,6 +30,22 @@ public function findDepartmentById($id)
         return $departments;
     }
 
+    public function findDepartmentByShortName($short_name)
+    {
+        $departments = $this->getEntityManager()->createQuery('
+            SELECT d
+            FROM AppBundle:Department d
+            WHERE lower(d.short_name) = lower(:short_name)
+        ')
+        ->setParameter('short_name', $short_name)
+        ->getResult();
```

Comment:
Forslag: Sjekke om department finnes i AdmissionController i stedet for her.
Da kan `getResult()` byttes med `getOneOrNullResult()`, så sjekke om `department == null` i controlleren.